flink-runtime/src/test/java/org/apache/flink/runtime/operators/MapTaskTest.java

```diff
@@ -38,8 +37,7 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-@SuppressWarnings("deprecation")
-public class MapTaskTest extends DriverTestBase<GenericCollectorMap<Record, Record>> {
+public class MapTaskTest extends DriverTestBase<FlatMapFunction<Record, Record>> {
```

Comment:
Small nit: should probably rename this to ``FlatMapTaskTest`` since now it is actually using ``flatMap`` function instead?